Menus/MainMenu/Scripts/MainMenuSelectable.cs

```diff
@@ -0,0 +1,73 @@
+﻿#if UNITY_EDITOR
+using System;
+using UnityEditor.Experimental.EditorVR;
+using UnityEngine;
+using UnityEngine.UI;
+using UnityEditor.Experimental.EditorVR.Core;
+
+#if INCLUDE_TEXT_MESH_PRO
+using TMPro;
+#endif
+
+[assembly: OptionalDependency("TMPro.TextMeshProUGUI", "INCLUDE_TEXT_MESH_PRO")]
+
+namespace UnityEditor.Experimental.EditorVR.Menus
+{
+    abstract class MainMenuSelectable : MonoBehaviour
+    {
+        protected Selectable m_Selectable;
+
+        [SerializeField]
+        HapticPulse m_ClickPulse;
+
+        [SerializeField]
+        HapticPulse m_HoverPulse;
+
+#if INCLUDE_TEXT_MESH_PRO
+        [SerializeField]
+        protected TextMeshProUGUI m_Description;
+#endif
+
+#if INCLUDE_TEXT_MESH_PRO
```

Comment:
Group these fields in the same #ifdef. I know this was my code but I did it wrong :)